lib/packages/echo.rb

```diff
@@ -0,0 +1,10 @@
+# A simple echo command for shellrb
+class Echo
+  def self.main (params)
+    if params[0] == '-n'
+      print params[1..(params.length - 1)].join(' ')
```

Comment:
this code is not very "DRY". you could do
`params[start..(params.length - 1)].join(' ')`
and just change value of start according to the conditions